test/puppetlabs/trapperkeeper/services/metrics/metrics_service_test.clj

```diff
@@ -61,20 +61,20 @@
            (is (= 200 (:status resp))))))
 
       (testing "register should add a metric to the registry"
-        (let [svc (app/get-service app :MetricsService)
-              registry (metrics-protocol/get-metrics-registry svc "pl.foo.reg")]
-          (metrics/register registry
-                            (metrics/host-metric-name "localhost" "foo")
-                            (metrics/gauge 2))
-          (let [resp (http-client/get
-                      (java.net.URLDecoder/decode
-                       (str "http://localhost:8180/metrics/v1/mbeans/"
-                            "pl.foo.reg:name=puppetlabs.localhost.foo")))
+        (doseq [[domain metric] [["pl.test.reg" "foo"] [:pl.test.reg "bar"]]]
```

Comment:
As a general practice, I'm against using `doseq`s in tests with the same assertions, because it becomes hard to tell under which condition the test failed. I would prefer to see a function wrapping the response that takes a domain, and then separate assertions for each domain with their own testing statements expressing why we're testing the two separately. But also this is already done as-is and also the PR has been open for a while and this is kind of a nitpick, so (shrug).